src/drive/targets/public/index.jsx

```diff
@@ -92,7 +92,6 @@ const init = async () => {
     cozyURL: cozyUrl,
     token: sharecode
   })
-
   const polyglot = initTranslation(dataset.locale, lang =>
```

Comment:
we can't share an encrypted folder for now?

Reply:
No target mobile update ?

Reply:
No we can't for sharing. Mobile should be added indeed